src/OEBPS/Chapter29.xhtml

```diff
@@ -76,7 +76,7 @@
             <p>Luna, undeterred, walked up to Mrs. Longbottom and looked up at the angular woman. “Hello, Mrs. Longbottom, it’s good to see you again,” she said calmly and held out her hand.</p>
             <p>Neville’s grandmother didn’t react, though Harry thought he saw a slight tightening around her eyes. “Neville,” she snapped.</p>
             <p>“Coming, Gran,” Neville said, making a great display of turning his trunk around. But once his grandmother’s back was turned to leave, he shrugged apologetically at the rest of them, looking rather confused.</p>
-            <p>Luna took advantage of the moment to step forward and give Neville a brief hug. He patted her shoulder twice with his free hand, but otherwise stood stock still. The blonde girl whispered something in his ear before she let got and quickly stepped back. Neville nodded once, then hefted the end of his trunk and set off after his grandmother.</p>
+            <p>Luna took advantage of the moment to step forward and give Neville a brief hug. He patted her shoulder twice with his free hand, but otherwise stood stock still. The blonde girl whispered something in his ear before she let go and quickly stepped back. Neville nodded once, then hefted the end of his trunk and set off after his grandmother.</p>
```

Comment:
:+1:

Reply:
Just to note, I've done a quick search for any other instances of "let got" and didn't find anyway.